### PR TITLE
Kotlin support to the plugin added

### PR DESCRIPTION
**Pull request by akbhatta**
*Issue #, if available:* NA

*Description of changes:*
Other changes:
Updated .idea project settings
Updated .gitignore
Added release notes creation script
Updated Gradle to 6.6

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: reports-scheduler/.codecov.yml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+codecov:
```
Comment (davidcui-amzn) [inline reports-scheduler/.codecov.yml:1]: Do we need the copyright notice in `yml` files?

File: reports-scheduler/config/checkstyle/checkstyle.xml [hunk 2]
```diff
@@ -0,0 +1,136 @@
+<?xml version="1.0"?>
```
Comment (davidcui-amzn) [inline reports-scheduler/config/checkstyle/checkstyle.xml:1]: Should this go below the copyright notice?

**Review by davidcui-amzn — APPROVED** [review]
A few questions about copyright notices missing from some files, but code LGTM

File: reports-scheduler/config/checkstyle/checkstyle.xml (hunk 2 above)
Comment (akbhatta) [reply reports-scheduler/config/checkstyle/checkstyle.xml:1]: XML requires the header to be on the top.

File: reports-scheduler/.codecov.yml (hunk 1 above)
Comment (akbhatta) [reply reports-scheduler/.codecov.yml:1]: added

**Review by abbashus — APPROVED** [review]

**Pull request merged**